duckduckgo/duckduckgo-privacy-extension
# Get script injection working for Chrome MV3 builds of the extension

Chrome Manifest V3 introduces the new scripting API[1] that can inject
scripts directly into the "main world" of a website. Along with that,
Manifest V3 prevents the old pattern of injecting a script element
into the DOM. Adjust to those changes here, to get our injected
protections working again.

1 - https://developer.chrome.com/docs/extensions/reference/scripting

**Reviewer:** @jonathanKingston 

## Steps to test this PR:
1. Navigate to the [storage blocking test page](https://privacy-test-pages.glitch.me/privacy-protections/storage-blocking/).
2. Click "Store data", then "Retrieve data", expand the results, and note which are blocked.
3. Build MV3 extension: npm install && npm run dev-chrome-mv3
4. Install that in Chrome.
5. Load the test page again and retry the test in step 2.
6. Ensure that more third party cookies are blocked.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
